index.html

```diff
@@ -4,9 +4,13 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="docs/logo.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Fluxo Ágil</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&family=Rubik:wght@600&display=swap" rel="stylesheet">    <title>Fluxo Ágil</title>
```

Comment:
Deixa o title na em uma linha separada

```suggestion
    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&family=Rubik:wght@600&display=swap" rel="stylesheet">    
    <title>Fluxo Ágil</title>
```